### ISSUE TITLE
2.0RC5:   java.lang.IllegalStateException - com.owncloud.android.ui.preview.PreviewImageFragment.onActivityCreated

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/android#1456]
```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2762)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:2848)
  at android.app.ActivityThread.-wrap12 (ActivityThread.java)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1552)
  at android.os.Handler.dispatchMessage (Handler.java:102)
  at android.os.Looper.loop (Looper.java:154)
  at android.app.ActivityThread.main (ActivityThread.java:6334)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:886)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:776)
Caused by: java.lang.IllegalStateException: 
  at com.owncloud.android.ui.preview.PreviewImageFragment.onActivityCreated (PreviewImageFragment.java:225)
  at android.support.v4.app.Fragment.performActivityCreated (Fragment.java:2257)
  at android.support.v4.app.FragmentManagerImpl.moveToState (FragmentManager.java:1367)
  at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState (FragmentManager.java:1569)
  at android.support.v4.app.FragmentManagerImpl.moveToState (FragmentManager.java:1636)
  at android.support.v4.app.FragmentManagerImpl.dispatchActivityCreated (FragmentManager.java:2977)
  at android.support.v4.app.FragmentController.dispatchActivityCreated (FragmentController.java:192)
  at android.support.v4.app.FragmentActivity.onStart (FragmentActivity.java:603)
  at android.support.v7.app.AppCompatActivity.onStart (AppCompatActivity.java:178)
  at com.owncloud.android.ui.activity.BaseActivity.onStart (BaseActivity.java:189)
  at com.owncloud.android.ui.activity.DrawerActivity.onStart (DrawerActivity.java:1273)
  at com.owncloud.android.ui.activity.FileActivity.onStart (FileActivity.java:188)
  at com.owncloud.android.ui.preview.PreviewImageActivity.onStart (PreviewImageActivity.java:173)
  at android.app.Instrumentation.callActivityOnStart (Instrumentation.java:1322)
  at android.app.Activity.performStart (Activity.java:6760)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2725)
```

**Comment by tobiasKaminsky** [comment]
225:  if (!getFile().isDown()) {
            throw new IllegalStateException("There is no local file to preview");
        }

A sane fix would be to show a good error message instead of silently crashing the whole app...

**Comment by AndyScherzinger** [comment]
fix provided via #1500
